### ISSUE TITLE
Update Color Tokens based on Community Feedback

### ISSUE THREAD

**Issue by muellernaumann** [porsche-design-system/porsche-design-system#4185]
Scope
---
With the release of v4 alpha, we are introducing a major update to our Color Token architecture to improve semantics, accessibility (WCAG 2.1), and dark mode support. Before the stable release, we want to align these changes with the community.

Notes  
---
Some disabled states looking to light, e.g. the primary button.

Acceptance criteria
---
1. Primitive Basis-Tokens (Fundament)
- [x] Primitive Token `dark gray-950` is changed to `#FBFCFF`
- [x] Primitive light `gray-50` fis changed to hsla(240, 10%, 95%, 1)
- [x] The underlying primitive (light gray-100a) for the frosted token in light has been updated to hsla(240, 5%, 70%, 0.148).
- [x] The underlying primitive (dark: gray-200a) for the frosted token in dark has been updated to hsla(240, 2%, 43%, 0.228).

2. Semantische Token-Definitionen (Frosted System)
- [x] New token frosted-strong is available and uses [light: gray-200a] [dark: gray-300a]
- [x] Tag have been updated for better contrast: 
-- Default State: Now uses frosted-strong (was frosted).
-- Hover State: Now uses frosted (was frosted-soft). 
- [x] Form element backgrounds now use frosted instead of frosted-soft.
- [x] Secondary buttons have been updated for better contrast: 
-- Default State: Now uses frosted-strong (was frosted).
-- Hover State: Now uses frosted (was frosted-soft).

3. Validierung & Status-Farben (Dark Mode & Logik)
- [x] The --p-color-info-frosted dark is updated to dark blue-200a
- [x] The --p-color-error-frosted dark is updated to dark red-200a
- [x] The --p-color-warning-frosted dark is updated to dark yellow-200a
- [x] The --p-color-success-frosted dark is updated to dark green-200a
- [x] No Hover on Validation States: When a form element (e.g., Input, Select, Checkbox) is in a validation state (Success, Error, Warning), hovering the element must not trigger a color change on the border. The border should be full tone.

Subtasks
---
- [ ] Task
